l10n_br_delivery_nfe/__manifest__.py

```diff
@@ -10,7 +10,12 @@
     "maintainers": ["mbcosta"],
     "website": "https://github.com/OCA/l10n-brazil",
     "version": "14.0.1.1.0",
-    "depends": ["l10n_br_nfe", "l10n_br_account", "l10n_br_delivery"],
+    "depends": [
+        "l10n_br_nfe",
+        "l10n_br_account",
+        "l10n_br_delivery",
+        "product_net_weight",
```

Comment:
Se for pra manter a compatibilidade, sem adicionar a dependência é melhor remover isso.